app/views.py

```diff
@@ -0,0 +1,142 @@
+ #app/views.py
+
+
+
+from flask import Flask,request,jsonify,abort
+from flask import make_response
+from app   import app
+# """
+#             Sample of requests
+#             {
+#             'id': 1,
+#             'title': 'Computer will not start',
+#             'description': 'The computer beeps three times then goes off',
+#             'type': 'repair'
+#             },
+            
+#             {
+#             'id': 2,
+#             'title': 'Access to the data centre',
+#             'description': 'There are updates that need to be installed on the servers',
+#             'type': 'maintenance'
+#             },
+
+#             {
+#             'id': 3,
+#             'title': 'Computer keyboard not working',
+#             'description': 'I acidentally poured coffee on my keyboard',
+#             'type': 'repair'
+#             }
+#             """
+requests = []
+users = []
+
+
+@app.route('/maintenanceapp/api/v1/requests', methods=['GET'])
+def get_requests():
+
+    return jsonify({'requests':requests})
+
+@app.route('/maintenaneapp/api/v1/requests/<int:request_id>', methods=['GET'])
+def get_request(request_id):
+        request = [request for request in requests if request['id']==request_id]
+        if len(request) == 0:
+            abort(404)
+        return jsonify({'request':request[0]})
+
+@app.errorhandler(404)
+def request_not_found(error):
+        return make_response(jsonify({'error':'Not found'}),404)  
+
+@app.route('/maintenanceapp/api/v1/requests', methods=['POST'])
+def create_request():   
+        if not request.json or not 'title' in request.json:
+            abort(400)
+        if len(requests)==0:    
+            app_request = {
+
+                        'id': request.json['id']+1,
```

Comment:
The id is supposed to be auto-generated not passed in the request. Use len() and 
then add 1.

Reply:
Done

Reply:
You havent' changed it

Reply:
@zmwas I have pushed the changes now [commit](https://github.com/RuthNjeri/Maintenance-Tracker-Application/pull/11/commits/98d4f084574b8a3d0b4806e91a14f06c3ce7e324)